docs/xxidletuningcompactonidle.md

```diff
@@ -28,7 +28,11 @@
 
 This option controls garbage collection processing with compaction when the state of the OpenJ9 VM is set to idle.
 
-<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** This option applies only to Linux&reg; architectures when the Generational Concurrent (`gencon`) garbage collection policy is in use. This option is not effective if the object heap is configured to use large pages.
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** 
+1. This option is deprecated and will be removed in the future. 
+2. This option applies only to Linux&reg; architectures when the Generational Concurrent (`gencon`) garbage collection policy is in use.
+3. This option is not effective if the object heap is configured to use large pages.
+4. This option is not effective when XX:IdleTuningGcOnIdle is not set.
```

Comment:
Same as above - using `-XX:[+|-]IdleTuningGcOnIdle` instead of `XX:IdleTuningGcOnIdle`.

Reply:
@ashu-mehra Should it be "This option is not effective if XX:+IdleTuningGcOnIdle is not set?"

Reply:
This option is not effective if `-XX:+IdleTuningGcOnIdle` is not specified`.

Reply:
Changed accordingly.